internal/domain/human_phone.go

```diff
@@ -27,17 +39,16 @@ type PhoneCode struct {
 	Expiry time.Duration
 }
 
-func (p *Phone) IsValid() bool {
-	err := p.formatPhone()
-	return p.PhoneNumber != "" && err == nil
-}
-
-func (p *Phone) formatPhone() error {
-	phoneNr, err := libphonenumber.Parse(p.PhoneNumber, defaultRegion)
+func (p *Phone) Normalize() error {
+	if p == nil {
+		return caos_errs.ThrowInvalidArgument(nil, "PHONE-YlbwO", "Errors.User.Phone.Empty")
+	}
+	normalizedNumber, err := p.PhoneNumber.Normalize()
 	if err != nil {
-		return caos_errs.ThrowInvalidArgument(nil, "EVENT-so0wa", "Errors.User.Phone.Invalid")
+		return err
 	}
-	p.PhoneNumber = libphonenumber.Format(phoneNr, libphonenumber.E164)
+	// TODO: Let's stop mutating state
```

Comment:
Follow up issue created: https://github.com/zitadel/zitadel/issues/5412

Reply:
Please add the comment to the file directly

Reply:
done